test/manual/kerberos.test.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 const { MongoClient } = require('../../src');
 const chai = require('chai');
+const sinon = require('sinon');
```

Comment:
Since the manual tests don't use the same config as the general integration tests, we need to include this line: `chai.use(require('sinon-chai'));` to be able to use the assertions that are currently failing in the CI

Reply:
Updated.